compiler_gym/envs/loop_tool/service/loop_tool_compilation_session.py

```diff
@@ -0,0 +1,272 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Define the loop_tool environment."""
+import logging
+import time
+from functools import reduce
+from pathlib import Path
+from typing import Optional, Tuple
+
+import numpy as np
+
+import loop_tool_py as lt
+from compiler_gym.service import CompilationSession
+from compiler_gym.service.proto import (
+    Action,
+    ActionSpace,
+    Benchmark,
+    Int64List,
+    Observation,
+    ObservationSpace,
+    ScalarLimit,
+    ScalarRange,
+    ScalarRangeList,
+)
+
+
+class LoopToolCompilationSession(CompilationSession):
+    """Represents an instance of an interactive loop_tool session."""
+
+    compiler_version: str = "0.0.1"
```

Comment:
I think this might be out of date :) Perhaps something like this?
```suggestion
    compiler_version: str = pkg_resources.get_distribution('loop-tool-py').version
```